data/styles/clear/include/Roads_label.mapcss

```diff
@@ -331,17 +331,40 @@ line|z15-[aerialway=t-bar],
 line|z15-[aerialway=magic_carpet],
 line|z15-[aerialway=platter],
 line|z15-[aerialway=rope_tow],
-{text: name;text-color: @label_dark;text-halo-radius: 1;text-halo-opacity: 0.7;text-halo-color: @label_halo_light;text-position: line;}
-
-/* 10.3 Piste & Route 14-22 ZOOM */
-/*
-line[piste:type=downhill],
-line[piste:type=nordic],
-line[piste:type=sled]
-{color: @piste;opacity: 1;}
-line|z14[route=ski],
-line|z14[leisure=skiing]
-*/
+line|z15-[piste:type=downhill],
+line|z15-[piste:type=nordic],
+line|z15-[piste:type=sled],
+{font-size: 9; text: name; text-color: @label_dark; text-halo-radius: 1; text-halo-opacity: 0.7; text-halo-color: @label_halo_light; text-position: line;}
+
+line|z16-17[aerialway=cable_car],
+line|z16-17[aerialway=gondola],
+line|z16-17[aerialway=mixed_lift],
+line|z16-17[aerialway=chair_lift],
+line|z16-17[aerialway=drag_lift],
+line|z16-17[aerialway=j-bar],
+line|z16-17[aerialway=t-bar],
+line|z16-17[aerialway=magic_carpet],
+line|z16-17[aerialway=platter],
+line|z16-17[aerialway=rope_tow],
+line|z16-17[piste:type=downhill],
+line|z16-17[piste:type=nordic],
+line|z16-17[piste:type=sled],
```

Comment:
nit: z16- is easier to maintain :)

Reply:
yes, but it slows down styles processing (e.g. the processor will apply z16- rules to z19 even though they'll be overriden later by z18- declaration)

Reply:
@pastk wow, that's bad! Does it happen in runtime? If yes, can you please create a separate issue for it? If no, and the issue only happens when styles are generated, then it's not a problem.

Reply:
its at the styles generation stage only

Reply:
Then let's keep it easier for _people_, not for _machines_ :)